### PR TITLE
PREAPPS-4164: System sends multiple requests for snooze all or dismiss all

### PR DESCRIPTION
**Pull request by sachinpatel88**

### CODE REVIEW

File: src/schema/notifications.ts [hunk 1]
```diff
@@ -485,7 +485,6 @@ export class ZimbraNotifications {
 			itemsToWrite = data.getNewMail;
 		} catch (exception) {
 			itemsToWrite = [];
-			console.error(exception);
```
Comment (sachinpatel88) [inline src/schema/notifications.ts]: It's known behaviour so not required to console it. (Change suggested in PREAPPS-1860)
cc: @aayushvijayvergiya @itsmeprashant

Comment (silentsakky) [inline src/schema/notifications.ts]: any reason for removing this?

File: src/schema/schema.graphql [hunk 2]
```diff
@@ -2884,8 +2884,8 @@ type Mutation {
 	sendInviteReply(inviteReply: InviteReplyInput!): InviteReplyResponse
 	sendShareNotification(shareNotification: ShareNotificationInput!): Boolean
 	setMailboxMetadata(section: String, attrs: MailboxMetadataSectionAttrsInput!): Boolean
-	snoozeCalendarItem(appointment: SnoozeInput, task: SnoozeInput): Boolean,
-	dismissCalendarItem(appointment: DismissInput, task: DismissInput): Boolean,
+	snoozeCalendarItem(appointment: [SnoozeInput], task: SnoozeInput): Boolean,
```
Comment (silentsakky) [inline src/schema/schema.graphql]: should we do similar changes for tasks as well? if not very huge

Comment (sachinpatel88) [reply src/schema/schema.graphql]: Currently, Reminders are only implemented for appointments and not for Tasks.

**Pull request merged**